backend/src/entities/game.ts

```diff
@@ -131,6 +134,27 @@ export class Game {
     };
   }
 
+  /**
+   * Update the score for the people who have not answer the question
+   */
+  public setPlayersMissingScores(): void {
+    const players = this.roomManager.getPlayers();
+    for (let i = 0; i < players.length; i++) {
+      if (this.receivedAnswersForCurrentTrack.includes(players[i]) === false) {
```

Comment:
Did you check if this is working ? I am not sure, but wouldn't Javascript consider that players in `this.receivedAnswersForCurrentTrack` and players in `this.roomManager.getPlayers` are different objects ?
Maybe it would be safer to compare player ids, so we are sure about it.

Reply:
Since it has the same typing it can be compared.